app/controllers/api/v0/instances_controller.rb

```diff
@@ -5,6 +5,7 @@ class InstancesController < ApiController
 
       def show
         render json: {
+          context: ApplicationConfig["FOREM_CONTEXT"],
```

Comment:
Just curious: are we going to pre-set this value somewhere so that creators that are self installing get it by default? Or is there a default value that Cloud/Systems can override?

Reply:
Cloud/Systems sets it. It defaults to `selfhost` and it changes to `forem_cloud` for instances on the cloud.

Reply:
https://github.com/forem/selfhost/blob/main/inventory/example/setup.yml#L22